gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -1368,6 +1418,22 @@ def delete_network_function_user_config(self, user_config_data,
                                                       config_params,
                                                       'DELETE')
 
+    def update_network_function_user_config(self, user_config_data,
+                                            service_config, config_tag):
+        config_params = self.create_request_structure(user_config_data,
+                                                      service_config,
+                                                      config_tag)
+        self._update_params(user_config_data,
+                            config_params, operation='update')
+        LOG.info(_LI("Sending update heat config request to configurator "
+                     " with config_params = %(config_params)s") %
+                 {'config_params': config_params})
```

Comment:
General question,
In some logs we used , and in some other logs we used % for concatenating format strings. Will both the formats work ?